apps/etheroscope_web/lib/etheroscope_web/router.ex

```diff
@@ -8,6 +8,7 @@ defmodule EtheroscopeWeb.Router do
   scope "/", EtheroscopeWeb do
     pipe_through :api
 
-    get "/history/:contract_address/:method", ContractController, :history
+    get "/contracts/:contract_address/", ContractController, :contract
+    get "/contracts/:contract_address/history/:variable", ContractController, :history
```

Comment:
I would put history before contract_address.

Reply:
That's not restful

Reply:
suggest `/history` gives histories for all variables
suggest `/history?variables=[var1,var2]` gives histories for var 1 and var 2

Reply:
Sounds good